modules/videoio/src/cap_gstreamer.cpp

```diff
@@ -315,36 +319,70 @@ static void find_hw_element(const GValue *item, gpointer va_type)
 
 class GStreamerCapture CV_FINAL : public IVideoCapture
 {
+public:
+    struct StreamData
```

Comment:
> StreamData

This type is not really used in the current state (used once to replace `videoStream` / `audioStream` fields).